packages/daft/daft.0.0.2/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+name: "daft"
+maintainer: "francois.berenger@inria.fr"
+authors: "Francois Berenger"
+homepage: "https://github.com/UnixJunkie/daft"
+bug-reports: "https://github.com/UnixJunkie/daft/issues"
+dev-repo: "https://github.com/UnixJunkie/daft.git"
+license: "CeCILL-C"
+build: [
+  [make]
+]
+install: [
+  ["cp" "-t" bin "daft" "daft_ds" "daft_mds"]
+]
+remove: [
+  ["rm" "-f" "%{bin}%/daft" "%{bin}%/daft_ds" "%{bin}%/daft_mds"]
+]
+depends: [
+  "obuild"
```

Comment:
This is probably a `{build}` dependency unless you extend `obuild` and so require to link with it at run-time.